lib/venue.rb

```diff
@@ -1,2 +1,20 @@
 class Venue
+  attr_reader :name, :capacity
+  def initialize (name, capacity)
+    @name = name
+    @capacity = capacity
+    @patrons = []
+  end
+
+  def patrons
+    patrons = []
+  end
+
+  def add_patron(new_patron)
+    patrons << new_patron.new
```

Comment:
You do not need to call `.new` on the `new_patron` that you are shoveling into `patrons`.